templates/components/multiagent/python/app/examples/publisher.py

```
@@ -0,0 +1,40 @@
+
+from app.agents.single import FunctionCallingAgent
+from app.engine.tools import ToolFactory
+from llama_index.core.chat_engine.types import ChatMessage
+from llama_index.core.tools import FunctionTool
+
+
+def get_publisher_tools() -> Tuple[List[FunctionTool], str, str]:
+    tools = []
+    # Get configured tools from the tools.yaml file
+    configured_tools = ToolFactory.from_env(map_result=True)
+    if "document_generator" in configured_tools.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)